bot/exts/evergreen/latex.py

```diff
@@ -60,7 +66,12 @@ def _prepare_input(text: str) -> str:
     async def latex(self, ctx: commands.Context, *, text: str) -> None:
```

Comment:
Following the discussion in #dev-contrib I think we want to add a [max_concurrency](https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.max_concurrency) to this command. I'd suggest with 1 concurrent, the guild bucket, and wait=True.